test/rest_api/test_column_family.py

```diff
@@ -3,14 +3,12 @@
 # SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
 
 import os
-import pytest
-import sys
+
 import requests
-import threading
```

Comment:
I get that they might be useless imports, but why change them here? How are they relevant to your change?